packages/webapp/src/components/Finances/Transaction/ExpandedContent/LabourTable.jsx

```diff
@@ -0,0 +1,173 @@
+/*
+ *  Copyright 2023 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+import React, { useState, useMemo } from 'react';
+import PropTypes from 'prop-types';
+import styles from './styles.module.scss';
+import { useTranslation } from 'react-i18next';
+import EnhancedTable from '../../../Table/v2';
+import StateTab from '../../../RouterTab/StateTab';
+import { useCurrencySymbol } from '../../../../containers/hooks/useCurrencySymbol';
+
+export default function LabourTable({ transaction }) {
+  const { t } = useTranslation();
+  const currencySymbol = useCurrencySymbol();
+
+  const [activeTab, setActiveTab] = useState('employee');
+
+  const employeeTotalHours = useMemo(
```

Comment:
It's quite possible -- @antsgar do you know off the top of your head? -- that the summation of hours in the `EMPLOYEE` and `TASK_TYPE` arrays _must_ be the same, but I didn't trust my knowledge of task edge cases enough to write it that way.

Reply:
You could actually just read this from `transaction.amount`, which is the sum of the items amounts when the transaction is grouped by tasks and the number that shows up on the expandable item up top. I trusted that summing by task and by employee would give the same result so only did it, hope that's working correctly 😅

Reply:
Well `transaction.amount` is the labour cost sum, while this refers to the hour sum!

Reply:
Ah you're right, totally got confused there!